### PR TITLE
Release notes: April IDE updates

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Adding a release note to match the [release announcement](https://getdbt.slack.com/archives/C03SAHKKG2Z/p1682966622484569) for the April IDE changes.

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x]  "needs technical review" from @bossnunta 

Adding new pages (delete if not applicable):
~~- [ ] Add page to `website/sidebars.js`~~ not needed as this section is auto generated
- [x] Provide a unique filename for the new page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+
+* Suggest you invoke `dbt deps` when needed (as informed by `dbt-score`) in a warning message
+* Provide warning when you select models bu do not save them before clicking **Build** button or dbt invocation (dbt build/run/test) 
+* Show live previews of Markdown and CSV files in the IDE console
+* Enable a Duplicate File option in the File Tree menu
+* Display loading time when Previewing a model
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Display loading time when previewing a model
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: "April IDE updates and fixes"
+id: "apr-ide-updates"
+description: "Apr 2023 release note: We've enhanced the IDE by ..., and more."
+sidebar_label: "Update and fixes: IDE"
+tags: [Apr-30-2023, IDE]
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md:7]:
```suggestion
sidebar_position: 7
tags: [Apr-30-2023, IDE]
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continue to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+
+* Suggest you invoke `dbt deps` when needed (as informed by `dbt-score`) in a warning message
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Suggest you invoke `dbt deps` when needed (as informed by `dbt-score`) by using a warning message
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continue to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+
+* Suggest you invoke `dbt deps` when needed (as informed by `dbt-score`) by using a warning message
+* Provide warning when you select models bu do not save them before clicking **Build** button or dbt invocation (dbt build/run/test)
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Provide warning when you select models but do not save them before clicking **Build** button or dbt invocation (dbt build/run/test) 
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continue to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+
+* Suggest you invoke `dbt deps` when needed (as informed by `dbt-score`) by using a warning message
+* Provide warning when you select models but do not save them before clicking **Build** button or dbt invocation (dbt build/run/test) 
+* Show live previews of Markdown and CSV files in the IDE console
+* Enable a Duplicate File option in the File Tree menu
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Provide a Duplicate File option in the File Tree menu
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+title: "April IDE updates and fixes"
+id: "apr-ide-updates"
+description: "Apr 2023 release note: We've enhanced the IDE by ..., and more."
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
description: "Apr 2023 release note: We've enhanced the IDE by displaying load times when previewing models, showing live previews of Markdown and CSV file in the IDE, adding the ability to duplicate files in the File Tree, and more."
```

**Review by bossnunta — DISMISSED** [review]
looks good to me!

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continue to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+
+* Suggest you invoke `dbt deps` when needed (as informed by `dbt-score`) by using a warning message
+* Provide warning when you select models but do not save them before clicking **Build** button or dbt invocation (dbt build/run/test)
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* New warning message appears when you select models but don't save them before clicking **Build** or invoking dbt (like, dbt build/run/test). 
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continue to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+
+* Suggest you invoke `dbt deps` when needed (as informed by `dbt-score`) by using a warning message
+* Provide warning when you select models but do not save them before clicking **Build** button or dbt invocation (dbt build/run/test) 
+* Show live previews of Markdown and CSV files in the IDE console
+* Provide a Duplicate File option in the File Tree menu
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* The file tree menu now includes a Duplicate File option.
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+tags: [Apr-30-2023, IDE]
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continue to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+
+* Suggest you invoke `dbt deps` when needed (as informed by `dbt-score`) by using a warning message
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* New warning message suggests you invoke `dbt deps` when it's needed (as informed by `dbt-score`).
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+* Provide warning when you select models but do not save them before clicking **Build** button or dbt invocation (dbt build/run/test) 
+* Show live previews of Markdown and CSV files in the IDE console
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Previews of Markdown and CSV files are now available in the IDE console.
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+* Provide a Duplicate File option in the File Tree menu
+* Display loading time when previewing a model
+
+## Product refinements 
+
+* Enhance autocomplete experience which has been laggy in the past for users with large projects and implement a limit to max `manifest.json` for this feature
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Enhance autocomplete experience which has performed slowly for people with large projects and who implement a limit to max `manifest.json` for this feature
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+* Introduce pagination for invocation node summary view (displaying 100 nodes at a time)
+* Improve rendering for the Changes / Version Control section of the IDE
+* Update icons to be consistent in dbt Cloud
+* Add table support to markdown preview
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Add table support to the Markdown preview
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+* Add lineage tab back to seed resources in the IDE
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Add the lineage tab back to seed resources in the IDE
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+* Implement modal priority when you see multiple warning modals
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Implement modal priority when there are multiple warning modals
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+* Adjust the description for a complex command in the command palette
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Improve a complex command's description in the command palette
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+
+* File tree no longer collapses on first click when there's a project subdirectory defined
+* **Revert all** button now works as expected
+* CSV preview no longer fails with only one column
+* Cursor and scroll bar location are now persistent with their positions
+* `git diff` view now shows just change diffs and no longer shows full diff (as if file is new) until page refreshes
+* ToggleMinimap Command no longer runs another Command at the same time
+* `git diff` view no longer shows infinite spins in specific scenarios (new file, etc.)
+* File contents no longer get mixed up when using diff view and one file has unsaved changes
+* YML lineage now renders model without tests (in core v1.5+)
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* YML lineage now renders model without tests (in dbt Core v1.5 and above)
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+* Radio buttons for "Summary" and "Details" in the logs section now consistently update to show the accurate tab selection
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* Radio buttons for **Summary** and **Details** in the logs section now consistently update to show the accurate tab selection
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 18]
```diff
@@ -0,0 +1,45 @@
...
+* IDE no longer throws the console error: `Error: Illegal argument` and redirects to the `Something went wrong` page
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
* IDE no longer throws the console error `Error: Illegal argument` and redirects to the `Something went wrong` page
```

File: website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md [hunk 19]
```diff
@@ -0,0 +1,45 @@
+---
+title: "April IDE updates and fixes"
+id: "apr-ide-updates"
+description: "Apr 2023 release note: We've enhanced the IDE by displaying load times when previewing models, showing live previews of Markdown and CSV file in the IDE, adding the ability to duplicate files in the File Tree, and more."
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/09-April-2023/apr-ide-updates.md]:
```suggestion
description: "Apr 2023 release note: We've enhanced the IDE by displaying load times when previewing models, showing live previews of Markdown and CSV files, adding the ability to duplicate files in the File Tree, and more."
```

**Review by nghi-ly — COMMENTED** [review]
Amazing updates and fixes to dbt Cloud's IDE!! Provided suggs for better clarity.

**Review by nghi-ly — APPROVED** [review]
:shipit:

**Pull request merged**